src/components/general/Divider/Divider.scss

```diff
@@ -1,6 +1,6 @@
 .Divider {
   height: 1px;
   width: 100%;
-  background-color: #ddd;
-  margin: 0.5em 0;
+  background-color: var(--border-color-2);
+  //   margin: 0.5em 0;
```

Comment:
We can just remove unused/commented code

Reply:
Sure thing!